insitu/views/product_requirement.py

```diff
@@ -37,6 +37,33 @@ def get_success_url(self):
                             kwargs={'pk': self.kwargs['requirement_pk']})
 
 
+class ProductGroupRequirementAdd(ProtectedCreateView):
```

Comment:
Why not better make a BaseProductRequirementAdd and then inherit it in both ProductGroupRequirementAdd and ProductRequirementAdd. Then in both child views modify only what is specific: form, template_name, title etc